android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt

```diff
@@ -231,6 +231,15 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
     private fun handleNavigation(@IdRes itemId: Int): Boolean {
         binding.drawerLayout.closeDrawers()
 
+        when (itemId) {
+            R.id.entire_survey -> {
+                navController.navigate(R.id.chrome, Bundle().apply {
```

Comment:
How about adding global action?
https://developer.android.com/guide/navigation/navigation-global-action?hl=en

Reply:
@takahirom 
I didn't know that.
This is much better, and I fixed it.
Thanks!